### ISSUE TITLE
Support for chaining

### ISSUE THREAD

**Issue by sampeka** [Foliotek/Croppie#52]
This is a great library, would be even better with support for chaining :)

**Comment by thedustinsmith** [comment]
Which methods would you like to see chained?  Most of the methods return something at this point, so I don't see how we would chain.

**Comment by sampeka** [comment]
Sorry, just realised how unclear this was. I meant show the cropped changes in another div in real time. As an example: http://fengyuanchen.github.io/cropper/

**Comment by thedustinsmith** [comment]
So you'd just like a demo that demonstrates how to do that?  Or is there some functionality that's missing?  I know we don't fire the update event until the dragging has completed, so the "real-time" would be missing.

**Comment by sampeka** [comment]
A demonstration would be really helpful. I think these things tend to look better when the changes tracked are live rather than have a delay for feedback - but it's definitely not essential.

**Comment by thedustinsmith** [comment]
I agree.  At one point the demo page did have live changes, but having multiple of those got a little too busy.  Maybe one wouldn't hurt.

**Comment by Lenndev** [comment]
Just fixed this. :) though the live result is a bit behind because you have to wait for the promise.

here is what i did: (just some html and js to show the basic setup doesnt work in fiddle cause no croppie!!!) 
https://jsfiddle.net/Lenndev/9rjzgsmu/
